GruntTasks/navigation_task.js

```diff
@@ -0,0 +1,44 @@
+module.exports = function(grunt) {
+
+    grunt.registerTask('navigation:compile', 'Open Orchestra task to find template underscore and compile it', function () {
+        var config = grunt.config('application.config');
+
+        var navigationConfigs = [];
+        grunt.util._.each(config.application.bundles, function (value) {
+            navigationConfigs.push('web/bundles/' + value + '/config/navigation.json')
+        });
+        navigationConfigs = grunt.file.expand(navigationConfigs);
+
+        var json = {};
+        // merge config
```

Comment:
@alavieille : instead of putting comment, can you use little function like mergeConfig ou sortNavigation ? It will help to have a better readibility and code maintenance